### DISCUSSION TITLE
Report of a malicious repository

### DISCUSSION THREAD

**Discussion by shahkashyap** [community/community#63603] [Pull Requests]
### Select Topic Area

Show & Tell

### Body

I am writing to report a repository that contains a **trojan virus**. The repository is called **### _BLXStealer_** and it claims to be a password stealer from Google Account. However, when I downloaded and Extract it , it infected my computer with a trojan virus that stole my personal information and files.
I request you to take immediate action to remove this repository and warn other users about the potential danger. This repository violates your terms of service and poses a serious threat to the security and privacy of your users.

Here is the git repository Link : https://github.com/blxstealer/BLX-Stealer

**Comment by ldez** [comment]
Hello,

if you have a problem with a GitHub account contact the [GitHub support](https://support.github.com/request).

Note: the archive is built by GitHub, so you ran this tool to be infected.
A hidden Remote access trojan (RAT) was inside the code.


I reported the account to GitHub support with all the information but you have to do the same:
- Go to the user profile: https://github.com/blxstealer.
- Click on "Block or Report" at the bottom left of the page.
- And choose "Report Abuse" and follow the process.

You have learned a lesson: never run this kind of project on your computer, and always read the code before running it.

**Comment by overtube** [comment]
This repo: `https://github.com/openhardwaremonitor/openhardwaremonitor` when installing on system, using system package manager (that have chroot), after ending installation, post install executing hook that that placing on `/usr/share` directory, **144** `ssh` `root` **certificates**. Offcource code safe and program worked well, but post install activity make malicious, thing because not ask user for trust or not and just wilfully install root certificates.

**Reply by roke-julian-lockhart** [reply]
@overtube, that's irrelevant to the original report, and worth an issue at the repository. The repository is not malicious.

**Comment by tayl054** [comment]
We have discovered that a malicious bad actor has setup a fake repo and a fake MAC-OS binary file to fake out users and put a malware on their computer. 

Our Github Ticket ID: 2723902

The malcious actor setup the account at https://github.com/macos-support 
under repos called Sonymusic and Sony with SME.dmg setup as the fake payload.

If you have any questions about this attack please reach out to me using my Github profile

**Comment by hideandseekstheorder** [comment]
### Select Topic Area
### Show & Tell

**Body**
I am writing to report a repository that contains a trojan virus. The repository is called ### BLXStealer and it claims to be a password stealer from Google Account. However, when I downloaded and Extract it , it infected my computer with a trojan virus that stole my personal information and files.
I request you to take immediate action to remove this repository and warn other users about the potential danger. This repository violates your terms of service and poses a serious threat to the security and privacy of your users.

Here is the git repository Link : https://github.com/SysAdminAce83/CCleaner-Professional

**Reply by hideandseekstheorder** [reply]
https://www.virustotal.com/gui/file/0ecddd9233d9f22116e0531b111496d0a7d9cbaf51583f5a0a26711f0078ebdf/detection

**Comment by bemax11** [comment]
I am writing to report a repository that contains a trojan virus, Hosting cpanel attack. cURL Link: https://raw.githubusercontent.com/ghostuserx/culrtest/main/index-mainfile.txt

**Reply by ldez** [reply]
Hello,

If you have a problem with a GitHub account or a repository, contact GitHub support.

- Go to the user profile
- Click "Block or Report" at the bottom left of the page.
- Choose "Report Abuse" and follow the process.

https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam

**Comment by kiro5678** [comment]


**Reply by ldez** [reply]
Hello,

If you have a problem with a GitHub account or a repository, contact GitHub support.

- Go to the user profile
- Click "Block or Report" at the bottom left of the page.
- Choose "Report Abuse" and follow the process.

https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam

**Comment by mowwrik** [comment]
Body
I am writing to report a repository that contains a trojan virus. The repository is called RC7 and it almost got my pc. It was infecting and trying to delete files from my pc and got admin rights without my permission. Luckly I quickly launched my antivirus and stopped it processes. I somehow managed to cure my system but it still got some of my files and damaged my system. Please do something to this guy because I face this type of virus first time in my life. I got really scared :(

Here is the git repository Link : https://github.com/RC7Dev/RC7

**Reply by ldez** [reply]
Hello,

If you have a problem with a GitHub account or a repository, contact GitHub support.

- Go to the user profile
- Click "Block or Report" at the bottom left of the page.
- Choose "Report Abuse" and follow the process.

https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam

**Comment by Axel-Erfurt** [comment]
How long does it take for such reports to be processed and the infected files are removed?

**Comment by adrianweidig** [comment]
I am writing to report a Trojan Repo including WinPE inside the download. Download is via external file in README.md:

https://github.com/bambina928/VRising-tool/tree/main

**Comment by SpaceCoderPro** [comment]
Trojan:Win32/Pomal!rfn  - PersepolisDownloadManager.Persepolis

**Comment by ldez** [comment]
Please stop posting on this thread.

If you have a problem with a GitHub account or a repository, contact GitHub support.

- Go to the user profile
- Click "Block or Report" at the bottom left of the page.
- Choose "Report Abuse" and follow the process.

https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam

**Reply by roke-julian-lockhart** [reply]
@ldez, I fear you may be screaming into the clouds. This is one of the strangest places I've seen – obviously, everyone can write, but they cannot read.
